composer.json

```diff
@@ -21,11 +21,12 @@
 		"email": "dev@mention-me.com"
 	},
 	"require":     {
-		"php": "^7.4",
+		"php": "^7.4 || ^8.0" ,
 		"ext-memcached": "*",
 		"symfony/symfony": "^4.4",
```

Comment:
Possibly worth sticking 5.4 in here now?

Reply:
Could do - although not too worried about it. I don't know what might be incompatible.